client/hosting/staging-site/index.tsx

```diff
@@ -5,10 +5,10 @@ import {
 	redirectIfCurrentUserCannot,
 	redirectToHostingPromoIfNotAtomic,
 } from 'calypso/controller';
+import { handleHostingPanelRedirect } from 'calypso/hosting/server-settings/controller';
```

Comment:
As a follow-up (not in this PR) maybe we need to extract this logic somewhere. It's copy-pasted literally in each tab for now. 😂

Reply:
Yeah, good point :)